wcc-contentful/spec/wcc/contentful/rich_text_renderer_spec.rb

```diff
@@ -0,0 +1,723 @@
+                'value' => 'This is an '
+              },
+              {
+                'nodeType' => 'entry-hyperlink',
+                'data' => {
+                  'target' => {
+                    'sys' => {
+                      'id' => '6mbnFhDqoOWFFAaE5O1HD9',
+                      'type' => 'Link',
+                      'linkType' => 'Entry'
+                    }
+                  }
+                },
+                'content' => [
+                  {
+                    'nodeType' => 'text',
+                    'value' => 'entry link'
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+
+      it 'renders an <a> tag if the model has #href' do
+        store = double('store', find: {
+          'sys' => {
+            'id' => '6mbnFhDqoOWFFAaE5O1HD9',
+            'type' => 'Asset'
+          },
+          'fields' => {
+            'file' => {
+              'url' => '//images.ctfassets.net/abc123/asset.jpg'
+            }
+          }
+        })
+
+        allow(subject).to receive(:store).and_return(store)
+
+        allow(subject).to receive(:model_api).and_return(
+          double('model-api', new_from_raw:
+            double('page-model', href: '/some-page'))
+        )
+
+        expect(subject.call).to match_inline_html_snapshot <<~HTML
+          <div class="contentful-rich-text">
+            <p>This is an <a href="/some-page">entry link</a></p>
+          </div>
+        HTML
+      end
+    end
+
+    context 'with an embedded-asset-block' do
+      let(:content) {
+        [
+          {
+            'nodeType' => 'embedded-asset-block',
+            'data' => {
+              'target' => {
+                'sys' => {
+                  'id' => '6mbnFhDqoOWFFAaE5O1HD9',
+                  'type' => 'Link',
+                  'linkType' => 'Asset'
+                }
+              }
+            }
+          }
+        ]
+      }
+
+      it 'renders an <img> tag' do
+        store = double('store', find: {
+          'sys' => {
+            'id' => '6mbnFhDqoOWFFAaE5O1HD9',
+            'type' => 'Asset'
+          },
+          'fields' => {
+            'title' => 'John Smith, Sr Pastor',
+            'file' => {
+              'url' => '//images.ctfassets.net/abc123/asset.jpg'
+            }
+          }
+        })
+
+        allow(subject).to receive(:store).and_return(store)
+
+        expect(subject.call).to match_inline_html_snapshot <<~HTML
+          <div class="contentful-rich-text">
+            <img src="//images.ctfassets.net/abc123/asset.jpg" alt="John Smith, Sr Pastor">
+          </div>
+        HTML
+      end
+    end
+  end
+
+  context 'with action view', rails: true do
+    before do
+      require 'wcc/contentful/action_view_rich_text_renderer'
+    end
+
+    subject { WCC::Contentful::ActionViewRichTextRenderer.new(document) }
+
+    it_behaves_like 'rich text renderer'
+  end
+
+  # TODO: nokogiri implementation?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/watermarkchurch/wcc-contentful/blob/d3d02e3857247983fbdaccc9cacb876d7b0e5bba/wcc-contentful/spec/wcc/contentful/rich_text_renderer_spec.rb#L722"></span>TODO added in <a href="https://github.com/watermarkchurch/wcc-contentful/blob/d3d02e3857247983fbdaccc9cacb876d7b0e5bba/wcc-contentful/spec/wcc/contentful/rich_text_renderer_spec.rb">wcc-contentful/spec/wcc/contentful/rich_text_renderer_spec.rb</a> - is there a card associated with that?</td>
    </tr>
  </tbody>
</table>